### DISCUSSION TITLE
Welcome to Til Discussions!

### DISCUSSION THREAD

**Discussion by cleberzavadniak** [til-lang/til#11] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * **Be excellent to each other**.

**Comment by dumblob** [comment]
May I still write proposals/ideas to the issue tracker as I did with so far? If not, could you clarify the policy where to draw the line?

I myself don't use Discussions for nothing else then offloading an **existing** issue from the issue tracker in case the topic becomes too blurry. Thus I'm supporting the "issue first, discussion second and only if really necessary" rule :wink:.

**Reply by cleberzavadniak** [reply]
Just testing this feature. But if I would draw a line, I probably would prefer issues to be "closable". Things too open to be marked as "resolved" should be sent here.
